deploy/helm/cht-chart/templates/couchdb-n-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+{{- $root := . -}}
+{{- if .Values.couchdb.clusteredCouch_enabled }}
+{{- range $i, $e := until (int .Values.clusteredCouch.noOfCouchDBNodes) }}
+{{ $nodeNumber := add $i 1 }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cht.service: couchdb-{{ $nodeNumber }}
+  name: cht-couchdb-{{ $nodeNumber }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cht.service: couchdb-{{ $nodeNumber }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        cht.service: couchdb-{{ $nodeNumber }}
+    spec:
+      tolerations:
+      - key: {{ $root.Values.toleration.key }}
+        operator: {{ $root.Values.toleration.operator }}
+        value: {{ $root.Values.toleration.value | quote }}
+        effect: {{ $root.Values.toleration.effect }}
+      containers:
+      - name: cht-couchdb-{{ $nodeNumber }}
+        image: public.ecr.aws/medic/cht-couchdb:{{ $root.Values.cht_image_tag }}
```

Comment:
Same, I think the upgrade service should send the whole image tag.

Reply:
Addressed.